packages/lodestar/src/node/loggerOptions.ts

```diff
@@ -0,0 +1,134 @@
+import {IConfigurationModule} from "../util/config";
+
+import {LogLevel, ILoggerOptions, defaultLogLevel} from "../logger/interface";
+
+export interface IBeaconLoggerOptions {
+  chain: ILoggerOptions;
+  db: ILoggerOptions;
+  eth1: ILoggerOptions;
+  node: ILoggerOptions;
+  network: ILoggerOptions;
+  opPool: ILoggerOptions;
+  sync: ILoggerOptions;
+}
+
+export interface IValidatorLoggerOptions {
+  validator: ILoggerOptions;
+}
+
+export const BeaconLoggerOptions: IConfigurationModule = {
+  name: "beacon-logger",
+  description: "beacon chain logger options",
+  fields: [
+    {
+      name: "chain",
+      fields: [
+        {
+          name: "level",
+          description: "log level",
+          type: "string",
+          configurable: true,
```

Comment:
Those configs won't expose cli flags! it's missing [cli property](https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar/src/util/config.ts#L8).

That said naming is quite awkward. I would much prefer single flag `logLevel` with [custom processing function to parse input](https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar/src/util/config.ts#L7)

Reply:
:+1: I'll add the custom processing